leehyowonzero/3week/76502.py

```diff
@@ -0,0 +1,25 @@
+from collections import deque
+
+def check(s):
+    q = deque()
+    for el in list(s):
+        if(len(q) > 0 and (ord(el)-ord(q[-1]) == 1 or ord(el)-ord(q[-1]) == 2)):
+            q.pop()
+        else:
+            q.append(el)
+    if(len(q)):
```

Comment:
```suggestion
    if q:
```

q 안에 원소가 존재하는지 확인할 때, 이렇게도 적을 수 있습니다. List나 다른 자료구조에도 동일해요!!